content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -36,6 +35,9 @@ Users need to be aware of the following when using this feature:
 * Cloning is only supported within the same Storage Class.
     - Destination volume must be the same storage class as the source
     - Default storage class can be used and storageClassName omitted in the spec
+* Cloning can only be performed between two volumes that use the same `VolumeMode` setting.
```

Comment:
For L32 above, can you clarify that the source PVC doesn't have to be dynamically provisioned as long as it meets the other requirements here.